admin-ui/src/pages/accounts-page.tsx

```
@@ -45,14 +44,29 @@ import { MagicCard } from "@/components/ui/magic-card"
-type WindowPreset = "86400000" | "604800000"
+type WindowPreset = "86400000" | "604800000" | "this_month"
 
 type SortBy = "account" | "requests" | "errors" | "tokens" | "last_req"
 
 function sum(items: Array<number | undefined>): number {
   return items.reduce<number>((acc, x) => acc + (x ?? 0), 0)
 }
 
+function windowPresetToRange(
+  preset: WindowPreset,
+  nowMs: number
```

ESLint

[error] Insert `,`

(prettier/prettier)